code/package.json

```diff
@@ -10,6 +10,7 @@
     "eslint-plugin-jsx-a11y": "^6.3.1",
```

Comment:
There seems to be some problem when you are doing the npm install, because I see you have at least 3 different package-lock.json and 2 different package.json, when you should only have one of each I think? 👀
Make sure to be in code folder when doing the npm install.